suzieq/cli/sqcmds/command.py

```diff
@@ -447,6 +448,9 @@ def _gen_output(self, df: pd.DataFrame, json_orient: str = "records",
             for row in df.itertuples():
                 self._pager_print(row.config)
         else:
+            if (self.start_time and self.end_time) or self.view == "all":
+                if 'active' in df.columns:
+                    df['active'] = np.where(df.active, '+', '-')
```

Comment:
We should change this check in order to have consistency. With this code, if we do `interface show columns='namespace hostname active'` we see `active` columns with `False/True`. I think is better if we show also here `+/-`.
```suggestion
            if 'active' in columns:
                df['active'] = np.where(df.active, '+', '-')
```